src/app/admin/packingSlotsTable/PackingSlotActions.ts

```diff
@@ -0,0 +1,162 @@
+import supabase from "@/supabaseClient";
+import { DatabaseError } from "@/app/errorClasses";
+import { GridRowId } from "@mui/x-data-grid";
+import { PackingSlotRow } from "@/app/admin/packingSlotsTable/PackingSlotsTable";
+
+export interface DBPackingSlotData {
+    primary_key?: string;
+    name: string;
+    is_shown: boolean;
+    order: number;
+}
+
+export interface DataGridPackingSlotRow {
+    id: string;
+    name: string;
+    is_shown: boolean;
+    order: number;
+}
+interface packingSlotTableRowData extends DataGridPackingSlotRow {
```

Comment:
The interface should be capitalised like `PackingSlot...`.

I think we can also remove the `Data` suffix since it doesn't add much context to the interface name. What do you think?